zoo-css-challenge/style.css

```diff
@@ -30,7 +31,7 @@ section {
 }
 
 img {
-  border: 5px solid #8075ff;
+  border: none;
```

Comment:
Alternatively, we can remove the entire `img` rule because `img` elements by default don't have a border